editor/src/clj/editor/prefs_dialog.clj

```diff
@@ -100,6 +100,7 @@
   (cond-> [{:name  "General"
             :prefs [{:label "Load External Changes on App Focus" :type :boolean :key [:workflow :load-external-changes-on-app-focus]}
                     {:label "Open Bundle Target Folder" :type :boolean :key [:bundle :open-output-directory]}
+                    {:label "Open browser after `Build HTML5`" :type :boolean :key [:build :open-html5-build]}
```

Comment:
The rest of the preferences here use Title Cased Labels.

Reply:
👍 fixed